scripts/build

```diff
@@ -14,12 +14,13 @@ fi
 
 echo "The repo name is: " $2
 
-./StorageAdapter/run.sh "Storage Adapter tests"
-
-./Config/run.sh "Config tests"
 
 ./IoTHubManager/run.sh "IoT Hub Manager tests"
 
+./StorageAdapter/run.sh "Storage Adapter Tests"
```

Comment:
Does order matter?

Reply:
no, I changed it for debugging my taxis failures.